frontend/assets/eak-lettering.svg

```diff
@@ -0,0 +1 @@
.01v0.14322h-1.2051v-0.14322q0.10372-5e-3 0.17286-0.01 0.0741-5e-3 0.1136-0.0296 0.0444-0.0247 0.0642-0.079 0.0198-0.0593 0.0198-0.16792v-2.7411q0-0.12841-0.0395-0.1778-0.0395-0.0543-0.11854-0.0543-0.0741 0-0.18767-0.0148v-0.12347l0.7754-0.12347 0.0346 0.0296v2.3114h0.01q0.2667-0.39017 0.74577-0.9532 0.079-0.0938 0.13335-0.11854 0.0543-0.0247 0.1778-0.0247h0.50871v0.13829q-0.33091 0.0346-0.57785 0.21237-0.28152 0.20743-0.56303 0.53834-0.0346 0.0444-0.0346 0.0642 0 0.0692 0.3803 0.51365 0.38029 0.43956 0.51858 0.57291 0.21237 0.21237 0.42475 0.2667z"/><path d="m129.45 140.01q-0.29633 0.14817-0.58279 0.14817-0.12841 0-0.19755-0.0741-0.0692-0.079-0.12347-0.22225-0.44944 0.29633-0.7359 0.29633-0.31609 0-0.51364-0.19262-0.19756-0.19261-0.19756-0.5087 0-0.43956 0.45932-0.58773 0.14323-0.0346 0.52352-0.15804 0.38029-0.12841 0.46919-0.21731v-0.20744q0-0.22225-0.10371-0.3556-0.10372-0.13829-0.31609-0.13829-0.34572 0-0.50871 0.48401-0.0247 0.0741-0.0543 0.10866-0.0296 0.0296-0.10372 0.0296-0.0988 0-0.18768-0.0642-0.0889-0.0692-0.0889-0.16792 0-0.13829 0.10866-0.25189 0.11359-0.11853 0.27657-0.18274 0.16793-0.0642 0.32597-0.0938 0.16298-0.0346 0.28646-0.0346 0.39511 0 0.60254 0.14323t0.20743 0.51364v1.1952q0 0.13828 0.01 0.21731 0.0148 0.079 0.079 0.14816 0.0642 0.0692 0.18768 0.0692 0.0741 0 0.14817-0.0148zm-0.89888-0.61736v-0.69638q-0.0938 0.0593-0.28151 0.12841-0.18768 0.0642-0.32103 0.1136t-0.24201 0.16792q-0.10865 0.11359-0.10865 0.28151 0 0.22719 0.11359 0.37042 0.11854 0.13829 0.34079 0.13829 0.22719 0 0.36054-0.13829 0.13828-0.13829 0.13828-0.36548z"/><path d="m131.35 137.87q0 0.0988-0.0642 0.18768-0.0642 0.0889-0.16298 0.0889-0.0494 0-0.18274-0.0642-0.13335-0.0691-0.22225-0.0691-0.14817 0-0.21731 0.16792-0.0642 0.16792-0.0642 0.34078v1.1557q0 0.15311 0.0593 0.21731t0.16298 0.0642q0.10866-5e-3 0.24695 5e-3v0.14322h-1.3039v-0.14322q0.10372-5e-3 0.17286-0.01 0.0692-5e-3 0.1136-0.0296 0.0445-0.0247 0.0642-0.079 0.0198-0.0593 0.0198-0.16792v-1.5558q0-0.12347-0.0395-0.17286-0.0395-0.0494-0.11854-0.0543-0.0741-5e-3 -0.18768-0.0198v-0.12348l0.75072-0.12347 0.0593 0.39511q0.0543-0.0543 0.13829-0.12347 0.084-0.0691 0.16298-0.12841 0.079-0.0593 0.17287-0.0988 0.0938-0.0445 0.1778-0.0445 0.11359 0 0.18767 0.0642 0.0741 0.0642 0.0741 0.1778z"/><path d="m133.05 139.95q-0.34572 0.21731-0.63218 0.21731-0.26176 0-0.40499-0.14817-0.14322-0.14816-0.14322-0.41486v-1.7187h-0.3556v-0.19755h0.37535l0.30127-0.6223h0.14323v0.6223h0.63712v0.19755h-0.63712v1.5311q0 0.49389 0.32597 0.49389 0.18768 0 0.35066-0.0642z"/><path d="m135.44 138.62q0 0.18768-0.19755 0.18768h-1.4224v0.0543q0 1.052 0.72602 1.052 0.46919 0 0.71613-0.52352l0.14817 0.079q-0.23707 0.69638-0.98284 0.69638-1.1211 0-1.1211-1.2495 0-0.57291 0.28646-0.92851 0.28646-0.36054 0.84949-0.36054 0.47413 0 0.73589 0.26176t0.26176 0.73096zm-0.51364-0.15805q0-0.28151-0.12347-0.46919-0.12347-0.19262-0.39017-0.19262-0.29634 0-0.4445 0.25189-0.14323 0.25188-0.14323 0.56797h0.93345q0.16792 0 0.16792-0.15805z"/></g></g></svg>
```

Comment:
Can't we just use the Inkscape SVG?

Reply:
Well, no, as the Inkscape SVG has its text not converted to a single path yet. So if we wanted e.g. to swap the fonts, we can easily adjust the inkscape svg and convert it to a single path again...

Reply:
Ah, okay 👍🏽